lib/snowplow_ruby_duid/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module SnowplowRubyDuid
-  VERSION = '1.1.0'
+  VERSION = '1.1.1'
```

Comment:
TBH, I don't see any reason to bump a version of a gem when we don't change anything in the lib or test.

Reply:
> > > @sparrovv Not sure how this gem is pushed to nexus or gemstash...My update is not important but we should keep up to date...somehow. Maybe add instructions to README or add jenkins pipeline.
> > 
> > 
> > This gem is public and published to rubygems. https://rubygems.org/gems/snowplow_ruby_duid
> 
> We can add a github action to push it, but need the owner of the gem to provide credentials.

Good idea.  We can do that in the same way as Twiglet 👍

Reply:
The reason for bumping is that it doesn't push otherwise. But since there's no push here, that could be fine. Don't know.